core/modules/user/user.module

```diff
@@ -1630,6 +1636,13 @@ function user_menu_alter(&$items) {
   }
 }
 
+/**
+ * Return the user page tabs setting.
+ */
+function user_login_appearance_setting() {
```

Comment:
Why do we need a function for this? We could use `config_get('system.core', 'user_login_appearance_setting');` right where we need it. :wink:

Reply:
Looks neater to me. Happy to change.